tests/AutoDocControllerTest.php

```diff
@@ -12,18 +12,24 @@ class AutoDocControllerTest extends TestCase
     use PHPMock;
 
     protected static array $documentation;
-    protected static string $localDriverFilePath;
+    protected static string $baseFile;
 
     public function setUp(): void
     {
         parent::setUp();
 
-        self::$localDriverFilePath ??= __DIR__ . '/../storage/documentation.json';
+        $documentationDirectory = config('auto-doc.drivers.local.directory');
+
+        self::$baseFile ??= $documentationDirectory.'/documentation.json';
         self::$documentation ??= $this->getJsonFixture('tmp_data');
 
-        file_put_contents(self::$localDriverFilePath, json_encode(self::$documentation));
+        if (!is_dir(storage_path($documentationDirectory))) {
+            mkdir(storage_path($documentationDirectory));
+        }
+
+        file_put_contents(storage_path(self::$baseFile), json_encode(self::$documentation));
```

Comment:
as self::$baseFile using onky here - no reasons to make it static, let's use simple variable instead of